ccp/src/cu/proving_thread/sync/state.rs

```diff
@@ -91,15 +93,16 @@ impl RandomXJob {
                 log::info!("proving_thread_sync: found new golden result hash {result_hash:?}\nfor local_nonce {:?}", self.local_nonce);
 
                 let proof = self.create_golden_proof(result_hash);
-                to_utility.send_proof(core_id, proof)?;
+                to_state_operating.send_proof(proof)?;
+                to_stats_collecting.send_proof(core_id)?;
```

Comment:
It seems that using `state_blabla' and `stats_blabla` was a mistake.  They often go in pairs, and here the distinction is on the 8th! character.

@mikevoronov WDYT?

Reply:
The prefix added makes a lot difference. I don't see how one can mix them up.

Reply:
Absolutely. But it is an extra work for tired eyes.

Reply:
I also think that it's better to find another name for `StateOperating` thread, but can't come up with anything more or less short.